src/main/java/sirius/biz/translations/MultiLanguageStringProperty.java

```diff
@@ -225,14 +224,7 @@ protected Object transformFromMongo(Value object) {
     @Override
     @SuppressWarnings("unchecked")
     protected Object transformToMongo(Object object) {
-        List<Document> texts = new ArrayList<>();
-        ((Map<String, String>) object).forEach((language, text) -> {
-            Document doc = new Document();
-            doc.put(LANGUAGE_PROPERTY, language);
-            doc.put(TEXT_PROPERTY, text);
-            texts.add(doc);
-        });
-        return texts;
+        return Json.convertFromMap((Map<String, String>) object);
```

Comment:
das geht so nicht, siehe gebrochene tests. in line247 könntest du das frommap nutzen

Reply:
Kp wie ich da drauf gekommen bin ... danke